plugins/inputs/docker_log/README.md

```diff
@@ -0,0 +1,63 @@
+# Docker Log Input Plugin
+
+The docker log plugin uses the Docker Engine API to get logs on running
```

Comment:
@prashanthjbabu We should make clear here that this only works with Docker daemons using `json-file` and `journald` as their backend.

See: https://github.com/moby/moby/blob/8391a667d6f04dd02d60d15c090ec4a2674f523c/api/swagger.yaml#L5093

Reply:
thanks for your comment! this makes sense! I've added it to the readme file!